times_reader/transforms.py

```diff
@@ -862,16 +1071,33 @@ def apply_fixups_table(table: datatypes.EmbeddedXlTable):
                         df.loc[index, ["CommName"]] = df[index][com_in_out]
 
         # Append _NRGI (energy input) to some cells in FLO_SHAR
-        i = (df["Attribute"].str.lower() == "share-i") & (
+        i = (df["Attribute"] == "SHARE-I") & (
             (df["LimType"] == "UP") | (df["LimType"] == "LO")
         )
         # TODO: looks like NRG may come from ~TFM_Csets
         df.loc[i, "Other_Indexes"] = df["TechName"].astype(str) + "_NRGI"
 
-        # TODO allow multiple columns in mapping
-        df["Attribute"] = df["Attribute"].str.replace(
-            "Share-I", "FLO_SHAR", case=False, regex=False
-        )
+        # Fill other indexes for some attributes
+        i = df["Attribute"].isin(["CEFF", "CEFFICIENCY", "CEFF-I", "CEFF-O"])
```

Comment:
Can we use the info from `aliases_by_attr` here?

Reply:
Let me add it as an issue. `aliases_by_attr` should probably be structured differently to use it both here and below. `"CEFF", "CEFFICIENCY", "CEFF-I", "CEFF-O","EFF", "EFFICIENCY"` are all kind of aliases of `ACT_EFF`, but with different defaults.